tests/notifications/sendReadClear.test.ts

```diff
@@ -302,6 +309,53 @@ describe('send/read/claim', () => {
       expect(exitCode).toBe(0);
       readNotificationMessages = JSON.parse(stdout);
       expect(readNotificationMessages).toHaveLength(0);
+      // Sending notifications to invalid nodeIds for outbox
+      const invalidNodeId = 'v53jhuvmfkc4tnngvor2op7c12jmohqa704pg3pcvrdui9j5kh900';
```

Comment:
How do you know this is invalid?

Reply:
i kinda just need a nodeId to black hole messages to that will always fail to send

Reply:
it's not an invalid node id per say, just a node id that is not associated with a live node

Reply:
i've used v0000000000000000000000000000000000000000000000000000 instead and renamed the var to `receivedNodeId`